client/src/components/AddVideoForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import Videos from "./Videos";
+import data from "../exampleresponse.json";
+
+function AddVideoForm() {
+  const defaultList = data;
```

Comment:
We expect all data is loaded from API instead of from test file

Reply:
Changes made to load data from API instead of test file. Please could you review when you get a chance please